CUETools.Codecs.FlaCuda/FlaCudaWriter.cs

```diff
@@ -156,14 +156,14 @@ public FlaCudaWriter(string path, Stream IO, AudioPCMConfig pcm)
 			sample_rate = pcm.SampleRate;
 			bits_per_sample = (uint) pcm.BitsPerSample;
 
-			// flake_validate_params
+			// Flake_validate_params

```

Comment:
Please undo. The lower case letter at the beginning of the method name should stay, like in other methods.
Furthermore `flake_validate_params` has been commented out throughout the project and is not relevant to this issue.